vllm/entrypoints/api_server.py

```diff
@@ -76,5 +76,5 @@ async def stream_results() -> AsyncGenerator[bytes, None]:
     uvicorn.run(app,
                 host=args.host,
                 port=args.port,
-                log_level="debug",
+                log_level="warning",
```

Comment:
Here we log only warning and critical errors for `uvicorn` dependency.

Reply:
Would it be better to omit this argument completely so that the `uvicorn` log level can be set by this own environment variable? See https://www.uvicorn.org/settings/